pkgs/applications/editors/spacevim/default.nix

```diff
@@ -0,0 +1,55 @@
+{ ripgrep, gitAndTools, fzf, makeWrapper, vim_configurable, vimPlugins, fetchFromGitHub, writeTextDir
+, stdenv, runCommandNoCC, remarshal, formats, spacevim_config ? import ./init.nix }:
+with stdenv;
+let
+  format = formats.toml {};
+  vim-customized = vim_configurable.customize {
+    name = "vim";
+    # Not clear at the moment how to import plugins such that
+    # SpaceVim finds them and does not auto download them to
+    # ~/.cache/vimfiles/repos
```

Comment:
Is there an upstream issue about this? Can you link it/create it? It should probably respect XDG_something.

Reply:
SpaceVim does respect XDG_CACHE_HOME https://github.com/SpaceVim/SpaceVim/blob/322c56a74d86df20ae58a46399bfbb2adcce9fc7/autoload/SpaceVim.vim#L216

The comment is more about the fact that SpaceVim will autodownload it's own plugins for vim rather than having my Nix derivation try and supply them.